Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
@snobu, do you mind update the code as I suggest above and give it a try to see what is the file name and content?Will be good to keep the full cursor value, since it will benefit for debugging.